app/controllers/applicants_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ApplicantsController < ApplicationController
+  def show
+    @applicant = Applicant.find(params[:id])
+    @applicant_pets = @applicant.pets
```

Comment:
You don't need to have this additional instance variable here. In your view you can call `@applicant.pets.each do`. This keeps your controller lean.